### ISSUE TITLE
Add MSRV into Cargo.toml

### ISSUE THREAD

**Issue by hombit** [Lokathor/bytemuck#143]
I have the same issue as in #142. Since the current version of the crate requires Rust 1.61, `rust-version` record of `Cargo.toml` is supported. Could you please add it there?

**Comment by Lokathor** [comment]
Unfortunately, that config key isn't conditionally available, so it doesn't actually work for this crate.

What MSRV specifically do you need?

It seems we'll have to split up the Arc support out of the main alloc support if enough people are really using the crate with a Rust that old.

**Comment by hombit** [comment]
Thank you for the answer, I was wrong proposing it, now I see that you need 1.60 for a single feature only. I use MSRV 1.57, but I've found that actually I don't use `bytemuck` as a "normal" dependency, I use it for tests and examples only.

**Comment by Lokathor** [comment]
Is it acceptable for the tests and examples to be built on a newer rust than your minimum rust? I admit it that sounds unusual.

I think that I made an error a few years ago when I picked 1.34 *specifically*, and I should have picked something like "Stable -10" or something, so that the entire crate could slowly accept updates. The current situation is certainly less than optimal because of it.

**Comment by hombit** [comment]
> Is it acceptable for the tests and examples to be built on a newer rust than your minimum rust? I admit it that sounds unusual.


Yes, sure, I just decided to not check MSRV build for everything but "normal" target.

Sorry for bothering you, I think "Stable -10" has a lot of sense, and I like this numerical constant, it gives a year of support =)
